security/try-pod-to-pod-encryption.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Try out the pod-to-pod encryption tech preview
+description: Try out enabling WireGuard for state-of-the-art cryptographic security between pods for Calico clusters.
+---
+
+### Big picture
+
+Enable WireGuard to secure pod-to-pod traffic in a {{site.prodname}} cluster.
+
+> **Warning!** Pod-to-pod encryption is a tech preview and should not be used in production clusters. The functionality is not supported with overlay networks (in particular, NodePort) and using NodePort from within a non host-networked will not work. It has had very limited testing and it will contain bugs (please report these on the Calico Users slack or GitHub). In addition, it does not support all the features of {{site.prodname}}.
+{: .alert .alert-danger }
+
+### Value
+
+{{ site.prodname }} supports WireGuard tunnels between nodes providing transport-level security for pod-to-pod traffic. WireGuard provides [formally verified](https://www.wireguard.com/formal-verification/) secure and [performant tunnels](https://www.wireguard.com/performance/) without any specialized hardware. For a deep dive in to WireGuard implementation, see [whitepaper](https://www.wireguard.com/papers/wireguard.pdf).
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- [Felix resource]({{ site.baseurl }}/reference/resources/felixconfig#felix-configuration-definition) with WireGuard configuration parameters.
+
+### Before you begin...
+
+- Verify the operating system(s) running on the nodes in the cluster [support WireGuard](https://www.wireguard.com/install/).
+
+    >**Note**: WireGuard in {{site.prodname}} does not support IPv6 at this time.
+   {: .alert .alert-info}
+
+### How to
+
+1. Install WireGuard on cluster nodes using these [instructions for your operating system](https://www.wireguard.com/install/).
+
+   >**Note**: Nodes that do not support WireGuard will not be secured by WireGuard tunnels, even if traffic running on the node to and from the pods goes to nodes that do support WireGuard.
+   {: .alert .alert-info}
+
+1. Enable WireGuard encryption across all the nodes using the following command.
+
+   ```
+   calicoctl patch felixconfiguration default --type='merge' -p '{"spec":{"wireguardEnabled":true}}'
+   ```
+
+   >**Note**: This above command can be used to change other WireGuard attributes. For a list of other WireGuard parameters and configuration evaluation, see the [Felix resource]({{ site.baseurl }}/reference/resources/felixconfig#felix-configuration-definition).
```

Comment:
The Note and paragraph on OpenShift should go under the step, not outside it.

Reply:
I moved it up in the last update as I was reading it. I can move it back if it makes sense with earlier format.